### PR TITLE
Add basic poly ert-test-example to README

### PR DESCRIPTION
**Pull request by Blunde1**
**Issue**
No standalone test-example for ert, without reservoir simulator, in README

**Approach**
Add poly_example to README

### CODE REVIEW

**Comment by ertomatic** [comment]
Can one of the admins verify this patch?

**Comment by sondreso** [comment]
Jenkins add to whitelist

File: README.md [hunk 1]
```diff
@@ -122,6 +122,16 @@ The C library files get installed into `res/.libs`, which is where the
...
 ## Example usage
 
+### Basic ert test
+To test if ert itself is working, go to `test-data/local/poly_example` and start ert by running `poly.ert` with `ert gui`
+```
+cd test-data/local/poly_example
+ert gui poly.ert
+````
+This opens up the ert graphical user interface.
+Finally, test ert by starting and successfully running the simmulation.
```
Comment (markusdregi) [inline README.md:132]:
```suggestion
Finally, test ert by starting and successfully running the simulation. 
```

**Review by markusdregi — COMMENTED** [review]
Looks good besides the small spelling error 👍🏻

The failing CI is a known problem discussed here: https://github.com/equinor/ert/issues/1231. It happens quite seldom and for the context of this PR, just try to restart the entire pipeline...

**Review by sondreso — APPROVED** [review]
LGTM!

**Pull request merged**